scripts/dts/extract_dts_includes.py

```diff
@@ -584,15 +584,24 @@ def yaml_traverse_inherited(node):
     :return: node
     """
 
+    if 'node_type' not in node.keys():
+        node['node_type'] = []
     if 'inherits' in node.keys():
-        if 'id' in node['inherits'].keys():
-            node['inherits']['node_type'] = node['inherits']['id']
-            node['inherits'].pop('id')
-        if 'inherits' in node['inherits'].keys():
-            node['inherits'] = yaml_traverse_inherited(node['inherits'])
-        dict_merge(node['inherits'], node)
-        node = node['inherits']
+        if isinstance(node['inherits'], list):
+            inherits_list  = node['inherits']
+        else:
+            inherits_list  = [node['inherits'],]
```

Comment:
isn't the ',' superfluous?

Reply:
For lists, yes.  (It's not for tuples.)  In any case, it's harmless.